src/unittest/test.h

```diff
@@ -31,8 +31,6 @@ _Pragma("warning( disable : 5105 )"); // 4251 warns when we export classes or st
 
 #ifdef _WIN32
 #define SHARED_OBJECT "dpp.dll"
-#elif __APPLE__
```

Comment:
why is this removed?

Reply:
Please revert this change. This was specifically added to fix unit tests breaking on macs.

This now breaks all our CIs for mac, along with any future unit test (if approved).